### DISCUSSION TITLE
Channel Logo Guidelines

### DISCUSSION THREAD

**Discussion by freearhey** [iptv-org/community#524] [Ideas]
I would like to add a list of recommendations for channel logos to the [iptv-org/database](https://github.com/iptv-org/database) repository [Contribution Guide](https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md).

Since finding a suitable logo for the channel is not always possible, this list contains only recommendations and should just help you in choosing a logo from several options.

1. If the channel logo has several versions, it is better to choose a color one.

    That way there is less chance of the logo blending in with the background.

    <img src="https://user-images.githubusercontent.com/7253922/235551685-34cbb982-078f-42b8-9fb3-2800afc80abe.png" width="600"/>

2. If the logo has changed over time, use the latest version.

    <img src="https://user-images.githubusercontent.com/7253922/235551700-e2ee6823-c6bc-4688-ad4c-081911703f42.png" width="600"/>

3. If the channel has several versions of the logo, it is better to use the one used on air.

    <img src="https://user-images.githubusercontent.com/7253922/235551795-4d89c2bc-f9eb-4ac2-88d4-2149a870776f.png" width="600"/>

    <img src="https://user-images.githubusercontent.com/7253922/235551738-3e6680ff-af77-4d22-af6d-cbaab7400b66.png" width="600"/>

4. If there is a transparent background around the logo is better to cut it.

    That way it's much easier to scale the logo later.
    
    <img src="https://user-images.githubusercontent.com/7253922/235551815-b9925ac5-85ac-458a-bf0b-a9d57eb2547d.png" width="600"/>

_UPDATE:_ Removed from the list are items that are mandatory requirements and are already listed in the [Contribution Guide](https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels).

If you have any suggestions, please leave them below.

After discussion, this list will be published [here](https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md).

**Comment by PopeyeTheSai10r** [comment]
All these suggestions are great. The only one I am not sure about is 4.

> If there is a transparent background around the logo **is better to cut it**.

I do agree with the statement. I am not sure if all contributors will want to use time to cut the background.

**Reply by freearhey** [reply]
Since this list is not a requirement, but only a recommendation, I think it will still be useful to have this paragraph in the list, given that most of the logos are already handled this way.
